packages/fiori/src/NotificationListItem.ts

```diff
@@ -166,15 +166,15 @@ const ICON_PER_STATUS_DESIGN = {
 class NotificationListItem extends NotificationListItemBase {
 	/**
 	* Defines if the `titleText` and `description` should wrap,
-	* they truncate by default.
+	* they wrap by default.
 	*
-	* **Note:** by default the `titleText` and `description`,
-	* and a `ShowMore/Less` button would be displayed.
-	* @default "None"
+	* **Note:** when the property is set ot "None" and the texts truncate
+	* a `More/Less` button would be displayed.
```

Comment:
I suggest:
Defines if the `titleText` and `description` should wrap when there is not enough space. The possible values are:
•	"Normal" (default) - the texts should wrap.
•	"None" – the texts should truncate, and a `More/Less` button should be displayed.